src/seedu/addressbook/commands/Command.java

```diff
@@ -39,12 +39,17 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
+    //public abstract CommandResult execute(){
+      //  throw new UnsupportedOperationException("This method should be implement in child classes");
+    //}
```

Comment:
Commented codes shouldn't be inside the commit; you have revision control to keep past important codes.